### PR TITLE
Devel

### PR DESCRIPTION
**Pull request by artur-trzesiok**

### CODE REVIEW

File: js/src/providers/threejs/objects/Volume.js [hunk 1]
```diff
@@ -12,6 +12,7 @@ var _ = require('lodash');
  */
 module.exports = function (config) {
     config.samples = config.samples || 512.0;
+    config.alpha_coef = config.alpha_coef || 50.0;
```
Comment (tgandor) [inline js/src/providers/threejs/objects/Volume.js]: What about 0.0?
(I know this would make it invisible, but it's a surprise when going from 1.0 to 0.0)

Comment (artur-trzesiok) [reply js/src/providers/threejs/objects/Volume.js]: this same with other parameters in every obj in k3d where 0.0 dosent make sense.

Comment (artur-trzesiok) [reply js/src/providers/threejs/objects/Volume.js]: Right now this approach is consist with whole code. If we decide to refactor it we will do it in all places at this same time.

Comment (tgandor) [reply js/src/providers/threejs/objects/Volume.js]: OK.

**Pull request merged**